addons/web/views/neutralize_views.xml

```diff
@@ -15,5 +15,8 @@
                 </span>
             </div>
         </xpath>
+        <xpath expr="//head">
+            <meta name="robots" content="noindex, nofollow" />
+        </xpath>
```

Comment:
Note about this diff, for tracking purpose:
- This view might get COW, directly or indirectly:
  - Directly: if someone were to write something in this view somehow (absolutely unlikely)
  - Indirectly: if someone cow the `web.layout`, which will cow also its children, which is something we often see
In the second case, we don't really care as if this `neutralize_banner` view is COW, it will still receive this update as the view will actually not have any arch diff (it was cow because its parent was).